lib/kubernetes-deploy/runner.rb

```diff
@@ -55,11 +55,12 @@ def self.with_friendly_errors
       exit 1
     end
 
-    def initialize(namespace:, environment:, current_sha:, context:, wait_for_completion:, template_folder: nil)
+    def initialize(namespace:, current_sha:, context:, wait_for_completion:, template_dir:)
       @namespace = namespace
       @context = context
       @current_sha = current_sha
-      @template_path = File.expand_path('./' + (template_folder || "config/deploy/#{environment}"))
+
+      @template_dir = Pathname.new(template_dir).realpath
```

Comment:
`File.expand_path` does look preferable given those examples, but really I'm happy as long as it is possible to write `template_dir: config/my/custom/path` (relative, no tilde) into your shipit.yml.